project/ws/app/src/lib/routes/home/routes/odcs-mapping/bulk-upload-odcs/bulk-upload-odcs.component.ts

```diff
@@ -0,0 +1,213 @@
+import { Component, OnInit, OnDestroy, AfterViewInit } from '@angular/core'
+import { MatSnackBar, PageEvent, MatDialog } from '@angular/material'
+import { HttpErrorResponse } from '@angular/common/http'
+import { ActivatedRoute } from '@angular/router'
+// tslint:disable-next-line
+import _ from 'lodash'
+import { Subject } from 'rxjs'
+import { takeUntil } from 'rxjs/operators'
+import { IBulkUploadDesignationList } from '../../designation/interface/interface'
+import { FileService } from '../../../../users/services/upload.service'
+import { UsersService } from '../../../../users/services/users.service'
+import { FileProgressComponent } from '../../users-view/file-progress/file-progress.component'
+import { VerifyOtpComponent } from '../../users-view/verify-otp/verify-otp.component'
+
+@Component({
+  selector: 'ws-app-bulk-upload-odcs',
+  templateUrl: './bulk-upload-odcs.component.html',
+  styleUrls: ['./bulk-upload-odcs.component.scss'],
+})
+export class BulkUploadOdcsComponent implements OnInit, OnDestroy, AfterViewInit {
+
+  lastUploadList: IBulkUploadDesignationList[] = []
+  private destroySubject$ = new Subject()
+  rootOrgId: any
+  bulkUploadFrameworkId = ''
+  configSvc: any
+  timeLeft = 10 // In Seconds
+  interval: any
+
+  showFileError = false
+  public fileName: any
+  fileSelected!: any
+  userProfile: any
+  fileUploadDialogInstance: any
+
+  sizeOptions = [10, 20]
```

Comment:
move to json, instead of hardcoding in component

Reply:
Added in config